float64 v_traj
